src/main/scala/firrtl/options/OptionParser.scala

```diff
@@ -0,0 +1,28 @@
+// See LICENSE for license details
+
+package firrtl.options
+
+import firrtl.{FIRRTLException, AnnotationSeq}
+import scopt.OptionParser
+
+trait TerminateOnExit { this: OptionParser[_] =>
+  var terminateOnExit = true
+  override def terminate(exitState: Either[String, Unit]): Unit =
+    if (terminateOnExit) sys.exit(0)
+}
+
+/** A modified [[scopt.OptionParser]] with mutable termination and additional checks
+  *
+  * @param name the name of the application
+  */
+trait DuplicateHandling extends OptionParser[AnnotationSeq] {
+  /* Check that no long or short option declarations are duplicated */
+  override def parse(args: Seq[String], init: AnnotationSeq): Option[AnnotationSeq] = {
+    val longDups = options.map(_.name).groupBy(identity).collect{ case (k, v) if v.size > 1 && k != "" => k }
+    val shortDups = options.map(_.shortOpt).flatten.groupBy(identity).collect{ case (k, v) if v.size > 1 => k }
```

Comment:
This seems to be responsible for:
```bash
An exception or error caused a run to abort: scopt.OptionDef.shortOpt()Lscala/Option; 
java.lang.NoSuchMethodError: scopt.OptionDef.shortOpt()Lscala/Option;
	at firrtl.options.DuplicateHandling$$anonfun$5.apply(OptionParser.scala:22)
	at firrtl.options.DuplicateHandling$$anonfun$5.apply(OptionParser.scala:22)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
	at scala.collection.immutable.List.foreach(List.scala:392)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.collection.mutable.ListBuffer.foreach(ListBuffer.scala:45)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:234)
	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
	at firrtl.options.DuplicateHandling$class.parse(OptionParser.scala:22)
	at firrtl.options.ExecutionOptionsManager$$anon$1.parse(ExecutionOptionsManager.scala:20)
	at firrtl.options.ExecutionOptionsManager.parse(ExecutionOptionsManager.scala:68)
	at firrtl_interpreter.Driver$.execute(Driver.scala:33)
	at firrtl_interpreter.DriverSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(DriverSpec.scala:21)
	at firrtl_interpreter.DriverSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(DriverSpec.scala:10)
	at firrtl_interpreter.DriverSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(DriverSpec.scala:10)
```

Reply:
False alarm. This was due to firrtl-interpreter using an old version of scopt.

Reply:
Possibly a stale scopt version? That option showed up in 3.7.0.